osvr/RenderKit/RenderManagerD3D11ATW.h

```diff
@@ -593,7 +519,8 @@ namespace osvr {
                         D3D11_BIND_RENDER_TARGET | D3D11_BIND_SHADER_RESOURCE;
                       textureDesc.CPUAccessFlags = 0;
                       textureDesc.MiscFlags = D3D11_RESOURCE_MISC_SHARED_KEYEDMUTEX;
-                      hr = m_D3D11device->CreateTexture2D(&textureDesc, NULL, &texture2D);
+                      ID3D11Texture2D* textureCopy = nullptr;
+                      hr = m_D3D11device->CreateTexture2D(&textureDesc, NULL, &textureCopy);
```

Comment:
We assign the buffer info's textureCopy field to this variable when it's created later on in this function. That field is then released in the RenderManagerD3D11ATW destructor.